doc/man/sconsign.xml

```diff
@@ -49,67 +49,58 @@
 
 
 <refsect1 id='description'><title>DESCRIPTION</title>
-<para>The
-<command>sconsign</command>
-command
-displays the contents of one or more signature database
-(<firstterm>sconsign</firstterm>)
-files used by the <command>scons</command> build tool.
+<para>
+Displays the contents of one or more
+<firstterm>sconsign files</firstterm>,
+the signature database files
+used by the <application>SCons</application> build tool.
 </para>
 
 <para>By default,
 <command>sconsign</command>
 dumps the entire contents of the
 sconsign file(s).
-Without the verbose option,
-each entry is printed in the following format:</para>
+Without options,
+individual dependency entries are printed in the following format:</para>
 
-<literallayout class="monospaced">
-file: signature timestamp length
-        implicit_dependency_1: signature timestamp length
-        implicit_dependency_2: signature timestamp length
+<screen>
+depfile: signature timestamp length
+        implicit_dependency_1: content_signature timestamp length
+        implicit_dependency_2: content_signature timestamp length
         ...
         action_signature [action string]
-</literallayout>
+</screen>
 
 <para><emphasis role="bold">None</emphasis>
-is printed
-in place of any missing timestamp, <firstterm>build signature</firstterm>
-(<emphasis role="bold">bsig</emphasis>),
-or <firstterm>content signature</firstterm>
+is printed in place of any missing timestamp,
+ <firstterm>content signature</firstterm>
```

Comment:
why drop build signature or change the wording from content signature?

Reply:
fixed - this went into the wrong file.

Reply:
I left out the "definition" bsig because it isn't used at all in the manpage, just the term "build action signature" is used now, csig is left in because it is used; the complete sentence without markup is:

> None is printed in place of any missing timestamp, content signature (csig) or build action signature values for any entry...